flex/paths.py

```diff
@@ -133,20 +133,21 @@ def match_path_to_api_path(path_definitions, target_path, base_path='', context=
     assert isinstance(context, collections.Mapping)
```

Comment:
(Arbitrary anchor)

Looks like the [failing test](https://github.com/pipermerriam/flex/blob/8419783f293632258b5ce15c0737e620f9271bd3/tests/validation/api_call/test_api_call_validation.py#L16) is because the [test schema has `basePath: v3`](https://github.com/pipermerriam/flex/blob/8419783f293632258b5ce15c0737e620f9271bd3/tests/schemas/httpbin.yaml#L4), but the test is hitting `/get` and not `/v3/get`.

Since `/v3/get` doesn't exist on httpbin, the best option I can think of is to remove the `basePath` from the test schema.  No `basePath` according to OAS means ["the API is served directly under the host"](http://swagger.io/specification/#swaggerBasePath), which seems accurate for httpbin.

What do you think about that option?